unittest/serializer/SerializerUnittest.cpp

```diff
@@ -128,20 +135,20 @@ TEST_F(SerializerUnittest, Write)
         fs.seekg(0, fs.end);
         int filelength = fs.tellg();
         fs.seekg(0, fs.beg);
-        ASSERT_EQ(sizeof(double)*jsize*ksize, filelength);
+        ASSERT_EQ(sizeof(float)*jsize*ksize, filelength);
 
         std::vector<char> rawdata(filelength);
         fs.read(rawdata.data(), filelength);
         fs.close();
 
         // Check data
-        const double* pNewData = reinterpret_cast<const double*>(rawdata.data());
+        const float* pNewData = reinterpret_cast<const float*>(rawdata.data());
```

Comment:
Why the change from double to float here? The data we are reading is double precision right?